src/test/scala/org/broadinstitute/dsde/firecloud/webservice/TrialApiServiceSpec.scala

```diff
@@ -141,29 +147,73 @@ final class TrialApiServiceSpec extends BaseServiceSpec with UserApiService with
-      Post(invalidPath, disabledUserEmail) ~> dummyUserIdHeaders(disabledUser) ~> trialApiServiceRoutes ~> check {
+      Post(invalidPath, disabledUserEmails) ~> dummyUserIdHeaders(dummy1User) ~> trialApiServiceRoutes ~> check {
         assert(!handled)
       }
     }
 
     "Attempting to enable a previously disabled user should return NoContent success" in {
-      Post(enablePath, disabledUserEmail) ~> dummyUserIdHeaders(disabledUser) ~> trialApiServiceRoutes ~> check {
+      Post(enablePath, disabledUserEmails) ~> dummyUserIdHeaders(dummy1User) ~> trialApiServiceRoutes ~> check {
         assertResult(NoContent, response.entity.asString) { status }
       }
     }
 
     "Attempting to enable a previously enabled user should return NoContent success" in {
-      Post(enablePath, enabledUserEmail) ~> dummyUserIdHeaders(enabledUser) ~> trialApiServiceRoutes ~> check {
+      Post(enablePath, enabledUserEmails) ~> dummyUserIdHeaders(dummy1User) ~> trialApiServiceRoutes ~> check {
+        assertResult(NoContent, response.entity.asString) { status }
+      }
+    }
+
+    "Attempting to disable a previously enabled user should return NoContent success" in {
+      Post(disablePath, enabledUserEmails) ~> dummyUserIdHeaders(dummy1User) ~> trialApiServiceRoutes ~> check {
+        assertResult(NoContent, response.entity.asString) { status }
+      }
+    }
+
+    "Attempting to disable a previously disabled user should return NoContent success" in {
+      Post(disablePath, disabledUserEmails) ~> dummyUserIdHeaders(dummy1User) ~> trialApiServiceRoutes ~> check {
         assertResult(NoContent, response.entity.asString) { status }
       }
     }
 
-    // TODO: Test user disabling
+    "Attempting to terminate a previously enabled user should return NoContent success" in {
+      Post(terminatePath, enabledUserEmails) ~> dummyUserIdHeaders(dummy1User) ~> trialApiServiceRoutes ~> check {
+        assertResult(InternalServerError, response.entity.asString) { status }
+      }
+    }
 
-    // TODO: Test user termination
+    "Attempting to terminate a previously disabled user should return NoContent success" in {
+      Post(terminatePath, disabledUserEmails) ~> dummyUserIdHeaders(dummy1User) ~> trialApiServiceRoutes ~> check {
+        assertResult(InternalServerError, response.entity.asString) { status }
+      }
+    }
+
+    "Attempting to terminate a previously enrolled user should return NoContent success" in {
+      Post(terminatePath, enrolledUserEmails) ~> dummyUserIdHeaders(dummy1User) ~> trialApiServiceRoutes ~> check {
+        assertResult(NoContent, response.entity.asString) { status }
+      }
+    }
+
+    "Attempting to terminate a previously terminated user should return NoContent success" in {
+      Post(terminatePath, terminatedUserEmails) ~> dummyUserIdHeaders(dummy1User) ~> trialApiServiceRoutes ~> check {
+        assertResult(NoContent, response.entity.asString) { status }
+      }
+    }
```

Comment:
could you replace this entire block with something a bit terser but that also covers all possible operations? I tried this and it seemed to work:
```
    val allManagerOperations = Seq(enablePath, disablePath, terminatePath)

    // define the operations that should succeed. All other operations should fail.
    val successes: Map[Seq[String], Seq[String]] = Map(
      disabledUserEmails -> Seq(enablePath, disablePath),
      enabledUserEmails -> Seq(enablePath, disablePath),
      enrolledUserEmails -> Seq(terminatePath),
      terminatedUserEmails -> Seq(terminatePath)
    )

    successes.foreach { case (targetUsers:Seq[String], operationPaths:Seq[String]) =>
      operationPaths.foreach { path =>
        s"Attempting $path on ${targetUsers.head} should return NoContent success" in {
          Post(path, targetUsers) ~> dummyUserIdHeaders(dummy1User) ~> trialApiServiceRoutes ~> check {
            assertResult(NoContent, response.entity.asString) { status }
          }
        }
      }
      val expectedFailPaths:Set[String] = allManagerOperations.toSet diff operationPaths.toSet
      expectedFailPaths.foreach { path =>
        s"Attempting $path on ${targetUsers.head} should return InternalServerError failure" in {
          Post(path, targetUsers) ~> dummyUserIdHeaders(dummy1User) ~> trialApiServiceRoutes ~> check {
            assertResult(InternalServerError, response.entity.asString) {
              status
            }
          }
        }
      }
    }
```